static/css/styles.css

```diff
@@ -611,6 +618,23 @@ button.menu-button + nav a {
     max-width: 20rem;
     margin: 0.2rem auto;
 }
+
+.profile-page main section:first-of-type h3 {
+    font-weight: 100;
+    font-size: 1.4rem;
+}
+
+.profile-page main section:nth-of-type(2) article{
```

Comment:
space before ` {` ;)

Reply:
yes, i made the correction it in the next PR